src/devtools/client/debugger/src/components/TestInfo/TestCase.tsx

```diff
@@ -111,8 +112,11 @@ export function TestCase({
                   </div>
                 ) : null}
               </div>
-            </button>
-          </div>
+            </div>
+            <div className="flex self-start">
+              <MaterialIcon>chevron_right</MaterialIcon>
+            </div>
```

Comment:
probably want to show this on hover. With these visual changes, can you share a loom / screenshot

Reply:
Fixed! 
https://user-images.githubusercontent.com/15959269/206016795-5bcbbe92-27ee-4047-aec9-b8cbe3b561a5.mov